src/controllers/authenticationController.ts

```diff
@@ -200,6 +207,90 @@ export default class AuthenticationController {
       sessionTrackingId
     );
 
+    const errorOrMaybeAssertionRef = this.lollipopParams.isLollipopEnabled
+      ? await this.sessionStorage.getLollipopAssertionRefForUser(user)
+      : E.right(O.none);
+
+    if (E.isLeft(errorOrMaybeAssertionRef)) {
+      return ResponseErrorInternal(
+        "Error retrieving previous lollipop configuration"
+      );
+    }
+
+    if (
+      this.lollipopParams.isLollipopEnabled &&
+      O.isSome(errorOrMaybeAssertionRef.right)
+    ) {
+      // Sending a revoke message for previous assertionRef related to the same fiscalCode
+      // This operation is fire and forget
+      this.lollipopParams.lollipopService
```

Comment:
maybe `await` is missing here

Reply:
This is a promise executed in fire and forget, calling the `.catch` method start the execution. If we use `await` the execution will be stopped.